scripts/screeps_rl_simulator_harness.py

```
@@ -4407,6 +4416,72 @@ def _require_command_success(smoke: Any, result: Any, phase: str) -> JsonObject:
+            "retryable": False,
+        }
+    return {
+        "attempt": attempt,
+        "returncode": _coerce_int(result.get("returncode")),
+        "elapsedSeconds": result.get("elapsed_seconds"),
+        "outputExcerpt": _safe_text(result.get("output_excerpt", ""), 500),
+        "retryable": _is_retryable_compose_setup_failure(result),
+    }
+
+
+def _is_retryable_compose_setup_failure(result: Any) -> bool:
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `result`

(ANN401)